server/homebrew.api.js

```diff
@@ -77,7 +77,7 @@ const api = {
 				if(req.account){
 					throw { ...accessError, message: 'User is not an Author', HBErrorCode: '03', authors: stub.authors, brewTitle: stub.title };
 				}
-				throw { ...accessError, message: 'User is not logged in', HBErrorCode: '04' };
+				throw { ...accessError, message: 'User is not logged in', HBErrorCode: '04', authors: stub.authors, brewTitle: stub.title };
```

Comment:
Adding these additional properties has broken in `invalid author` test; we'll need to update the expected test result in the API spec document to get the test to pass again.

Reply:
Whoops. I'll fix that real quick.

Reply:
Done - its resolved in [this commit](https://github.com/naturalcrit/homebrewery/pull/2892/commits/a7040e554ac2d3a7b144f0983b31c604697d2f16).